documentation/host/deployment.rst

```diff
@@ -22,17 +23,25 @@ On your own computer, ``flexmeasures run`` is a nice way to start FlexMeasures.
    # This file contains the WSGI configuration required to serve up your
    # web application.
    # It works by setting the variable 'application' to a WSGI handler of some description.
+   # The crucial part are the last two lines. We add some ideas for possible other logic.
 
    import os
-   from dotenv import load_dotenv
-
    project_home = u'/path/to/your/code/flexmeasures'
+   # use this if you want to load your own ``.env`` file.
+   from dotenv import load_dotenv
    load_dotenv(os.path.join(project_home, '.env'))
+   # use this if you run from source
+   if project_home not in sys.path:
+      sys.path = [project_home] + sys.path
+   # adapt PATH to find our LP solver if it is installed from source
+   os.environ["PATH"] = os.environ.get("PATH") + ":/home/seita/Cbc-2.9/bin"
```

Comment:
Should the solver path be a config setting?

Reply:
One can probably also supply a full path to FLEXMEASURES_LP_SOLVER